designsafe/apps/audit/tests.py

```diff
@@ -0,0 +1,110 @@
+from django.test import Client
+from django.contrib.auth.models import User
+from django.urls import reverse
+from django.db import Error as DatabaseError
+from unittest.mock import patch
+import pytest
+
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.fixture
+def authenticated_user():
+    """Create and return an authenticated user for testing"""
+    user = User.objects.create_user(username="testuser", password="testpass")
+    return user
+
+
+@pytest.fixture
+def authenticated_client(authenticated_user):
+    """Create a client with an authenticated user"""
+    client = Client()
+    client.login(username="testuser", password="testpass")
+    return client
```

Comment:
I think these already exist in conftest, so you just need to add them to your parameters for your tests.